apps/ui/src/composables/useConnectors.ts

```diff
@@ -62,6 +62,15 @@ const CONNECTOR_DETAILS: Record<ConnectorType, ConnectorDetail> = {
         icons: ['https://snapshot.box/favicon.svg']
       }
     }
+  },
+  'sequence-waas': {
+    info: {
+      name: 'Sequence Embedded Wallet',
```

Comment:
We can just call it "Sequence", embedded wallet is too technical, we will later add a label "Detected" on injected wallets so users wont confuse the embedded wallet and the Chrome extension
```suggestion
      name: 'Sequence',
```